sdk/src/Core/Sampling/TimeStamp.cs

```diff
@@ -94,7 +94,7 @@ internal bool IsAfter(TimeStamp timeStamp)
 
         private static decimal GetUnixSeconds(DateTime utcNow)

```

Comment:
Was this a bug?

Reply:
Yes. It was always returning the current time as unix seconds.
It is a helper method primarily called by SetStartTime(timestamp) and SetEndTime(timestamp)